src/buttonBlock.js

```diff
@@ -0,0 +1,36 @@
+export default class ButtonBlock { 
+
+  blockManager;
+
+  constructor(editor, opts = {}) {  
```

Comment:
I think the `opts` parameter is not needed and can be removed

Reply:
Removed @adiux